docs/developer/webui/webui_frontend.md

```diff
@@ -583,4 +583,4 @@ Errors for the time being.
 
 ## Components
 
-## Folder structure -->
\ No newline at end of file
+## Folder structure -->
```

Comment:
This diff looks a bit weird because as it looks identical, but I checked with `git diff --ignore-all-space` and it shows no diff, so it's only a safe whitespace change.